process/block/shardblock.go

```diff
@@ -1077,6 +1081,11 @@ func (sp *shardProcessor) receivedMetaBlock(metaBlockHash []byte) {
 		return
 	}
 
+	isMetaBlockHigherThanLastNotarized := metaBlock.GetNonce() > lastNotarizedHdr.GetNonce()+process.MaxHeadersToRequestInAdvance
```

Comment:
maybe call it metaBlockOutOfRange ? 
isMetaBlockHigherThanLastNotarized is misleading, as if metablock has a higher nonce than last notarized but in the accepted range you will set this variable to false

Reply:
Done